Mathlib/Topology/Algebra/ProperAction/Basic.lean

```diff
@@ -166,3 +167,42 @@ theorem properSMul_of_isClosedEmbedding {H : Type*} [Group H] [MulAction H X] [T
 /-- If `H` is a closed subgroup of `G` and `G` acts properly on `X`, then so does `H`. -/]
 instance {H : Subgroup G} [ProperSMul G X] [H_closed : IsClosed (H : Set G)] : ProperSMul H X :=
   properSMul_of_isClosedEmbedding H.subtype H_closed.isClosedEmbedding_subtypeVal fun _ _ ↦ rfl
+
+/-- The action `G ↷ G` by left translations is proper. -/
+@[to_additive
+/-- The action `G ↷ G` by left translations is proper. -/]
+instance [IsTopologicalGroup G] : ProperSMul G G where
+  isProperMap_smul_pair := by
+    let Φ : G × G ≃ₜ G × G :=
+    { toFun := fun gh ↦ (gh.1 * gh.2, gh.2)
+      invFun := fun gh ↦ (gh.1 * gh.2⁻¹, gh.2)
+      left_inv := fun _ ↦ by simp
+      right_inv := fun _ ↦ by simp }
+    exact Φ.isProperMap
+
+open MulOpposite in
+/-- The action `Gᵐᵒᵖ ↷ G` by right translations is proper. -/
+@[to_additive
+/-- The action `Gᵐᵒᵖ ↷ G` by right translations is proper. -/]
+instance [IsTopologicalGroup G] : ProperSMul Gᵐᵒᵖ G where
+  isProperMap_smul_pair := by
+    let Φ : Gᵐᵒᵖ × G ≃ₜ G × G :=
+    { toFun := fun gh ↦ (gh.2 * (unop gh.1), gh.2)
+      invFun := fun gh ↦ (op (gh.2⁻¹ * gh.1), gh.2)
+      left_inv := fun _ ↦ by simp
+      right_inv := fun _ ↦ by simp }
+    exact Φ.isProperMap
+
+example [IsTopologicalGroup G] {H : Subgroup G} [IsClosed (H : Set G)] : ProperSMul H G :=
+  inferInstance
```

Comment:
Did you really want to leave this in? I could see having it during development, but I don't think it needs to stay in the file. If you want to keep it, that's fine, but please add a comment explaining it.